load/InputFileset.py

```diff
@@ -138,16 +151,20 @@ def toString(self):
 	## This method is used instead of _setFilenames when a specific list of files has been provided
 	def _downloadSelectedFiles(self, fileList):
 		results = []
-		s3Client = AWSSession.shared().s3Client
-		directory = self.config.directory_upload_aws + self.filesetId
+		session = boto3.Session(profile_name = self.config.s3_aws_profile)
+		s3Client = session.client('s3')
+		directory = self.config.directory_upload_aws + self.filesetId + os.sep + self.filesetPath
```

Comment:
@bradflood I have changed it because the `directory_upload_aws` folder has a sub-directory with the `self.filesetPath`, so, we need to create the subdirectory before the file is saved in the local directory.

Reply:
we still have something wrong with our input data organization. There shouldn't be a need to create a new directory.

Reply:
@bradflood It has been removed.